changes/8849.refpix.rst

```diff
@@ -0,0 +1 @@
+Removed the interleve array and do interleving of noise and subtraction in-place to avoid creating 2 arrays of equal dimensions to data
```

Comment:
This file should probably have `emicorr` in its name rather than `refpix`

Reply:
Also a couple quick typo fixes:

```suggestion
Removed the interleaved noise array and do interleaving of noise and subtraction in-place to avoid creating 2 arrays of equal dimensions to data.
```

Reply:
done